org.mixxx.Mixxx.yaml

```
@@ -3,38 +3,53 @@ runtime: org.kde.Platform
 runtime-version: '5.15-23.08'
 sdk: org.kde.Sdk
 command: mixxx
+rename-icon: mixxx
+
+# Tag and mark as beta until 2.4 final release
+tags:
+  - beta
+desktop-file-name-suffix: ' (beta)'
+
 finish-args:
-  - --share=ipc
+  # Required for DJ controller support
+  - --device=all
+
+  # Wayland is used by default
   - --socket=wayland
-  - --socket=x11
-  # JACK support via PipeWire
-  - --filesystem=xdg-run/pipewire-0
-  # PulseAudio isn't actually helpful for Mixxx, but this allows access to ALSA
-  # devices too since https://github.com/flatpak/flatpak/pull/3663
-  - --socket=pulseaudio
-  # Only used for MusicBrainz lookups
+  - --socket=fallback-x11
+
+  # Used by X11 and MusicBrainz metadata lookups
+  - --share=ipc
   - --share=network
+
+  # Permit PipeWire and user music directory access
+  - --filesystem=xdg-run/pipewire-0
   - --filesystem=xdg-music
-  - --persist=.mixxx
-  - --device=all
-  - --metadata=X-DConf=migrate-path=/org/mixxx/Mixxx
-  # Required due to use of legacy GLWidget which does not work with Wayland QPA
-  - --env=QT_QPA_PLATFORM=xcb
-  - --env=LV2_PATH=/app/extensions/Plugins/lv2
-  # Required for battery widget
+
+  # PulseAudio is needed for ALSA device access
+  - --socket=pulseaudio
+
+  # Talk to UPower and RTKit services on the session bus
   - --system-talk-name=org.freedesktop.UPower
-  - --env=ALSA_CONFIG_PATH=
   - --system-talk-name=org.freedesktop.RealtimeKit1
 
-rename-icon: mixxx
+  # Define LV2 plugin location and ignore ALSA configuration
+  - --env=LV2_PATH=/app/extensions/Plugins/lv2
```

```suggestion
  # gnore ALSA configuration
```

LV2_PATH is properly set in the wrapper script.